plane/src/drone/wait_backend.rs

```diff
@@ -1,17 +1,28 @@
-use std::{net::SocketAddr, time::Duration};
+use std::{
+    net::SocketAddr,
+    time::{Duration, Instant},
+};
 use tokio::{
     io::{AsyncReadExt, AsyncWriteExt},
     net::TcpStream,
 };
 
```

Comment:
nit: remove the gap so that `cargo fmt` doesn't treat it as two blocks of imports